modules/move-etcd-different-disk.adoc

```diff
@@ -111,42 +125,29 @@ spec:
           WantedBy=multi-user.target graphical.target
         enabled: true
         name: restorecon-var-lib-etcd.service
-
 ----
 
-. Create the machine configuration by entering the following commands:
+. Log in to the cluster as a user with `cluster-admin` privileges and create the machine configuration:
```

Comment:
```suggestion
. Log in to the cluster as a user with `cluster-admin` privileges and create the machine configuration by running the following commands:
```
Per [OCP doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command)